CHANGELOG.md

```diff
@@ -1,5 +1,90 @@
 # Changelog of FOSSology
 
+### 4.0.0-rc1 (Dec 21st 2021)
+
+This release adds important corrections to
+[3.11.0](https://github.com/fossology/fossology/releases/tag/3.11.0)
+
+The release 4.0.0-rc1 introduces following major changes since
+[3.11.0](https://github.com/fossology/fossology/releases/tag/3.11.0):
+
+* Support Debian 11
+* Add bootstrap in fossology to beautify ui
+* Remove old gold files
+* Remove old log files
+* Read delimiters for clean text monkbulk
```

Comment:
So the custom delimiters are not limited to "clean text" button. They also effect how monkbulk works.

A better wording will be "Provide custom delimiters for monkbulk scan" imo.